Library/Homebrew/cmd/desc.rb

```diff
@@ -48,19 +52,32 @@ def desc
       :desc
     end
 
-    results = if search_type.nil?
+    if search_type.nil?
       desc = {}
-      args.named.to_formulae.each { |f| desc[f.full_name] = f.desc }
-      Descriptions.new(desc)
+      args.named.to_formulae_and_casks.each do |formula_or_cask|
+        if formula_or_cask.is_a? Formula
+          desc[formula_or_cask.full_name] = formula_or_cask.desc
+        else
+          description = formula_or_cask.desc.presence || Formatter.warning("[no description]")
+          desc[formula_or_cask.full_name] = "(#{formula_or_cask.name.join(", ")}) #{description}"
+        end
+      end
+      Descriptions.new(desc).print
     else
       query = args.named.join(" ")
       string_or_regex = query_regexp(query)
-      CacheStoreDatabase.use(:descriptions) do |db|
-        cache_store = DescriptionCacheStore.new(db)
-        Descriptions.search(string_or_regex, search_type, cache_store)
+      unless args.cask?
```

Comment:
How about doing a `elsif args.cask?` and `elsif args.formula?`

Reply:
This allows the absence of `--formula` and `--cask` to show results for both instead of neither, much like how `list` and `bump` work.

Reply:
Gotcha 👍🏻